src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -61,6 +61,7 @@
     public static final AttributeKey<String>        JENKINS_COMPUTER_NAME = AttributeKey.stringKey("jenkins.computer.name");
 
     public static final AttributeKey<String>        JENKINS_STEP_NODE_LABEL = AttributeKey.stringKey("jenkins.pipeline.step.node.label");
+    public static final AttributeKey<Long>          JENKINS_STEP_NODE_ALLOCATION_TIME = AttributeKey.longKey("jenkins.pipeline.step.node.allocation.time");
```

Comment:
I see benefits in having a span indicating the allocation duration:
* Spans are used to report durations
* All the automated anomaly detections based on span durations will work out of the box if the node allocation is captured as a dedicated span

Reply:
I just reverted the commit for the allocation time, so we can implement the span allocation duration instead